src/fireedge/src/modules/utils/webpack.config.prod.utils.js

```diff
@@ -0,0 +1,115 @@
+/* ------------------------------------------------------------------------- *
+ * Copyright 2002-2025, OpenNebula Project, OpenNebula Systems               *
+ *                                                                           *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may   *
+ * not use this file except in compliance with the License. You may obtain   *
+ * a copy of the License at                                                  *
+ *                                                                           *
+ * http://www.apache.org/licenses/LICENSE-2.0                                *
+ *                                                                           *
+ * Unless required by applicable law or agreed to in writing, software       *
+ * distributed under the License is distributed on an "AS IS" BASIS,         *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  *
+ * See the License for the specific language governing permissions and       *
+ * limitations under the License.                                            *
+ * ------------------------------------------------------------------------- */
+const moduleName = 'UtilsModule'
+const fs = require('fs-extra')
+const { parse: yamlToJson } = require('yaml')
+const path = require('path')
+const { ModuleFederationPlugin } = require('webpack').container
+const sharedDeps = require('../sharedDeps')
+const ExternalRemotesPlugin = require('external-remotes-plugin')
+const ONE_LOCATION = process.env.ONE_LOCATION
+const ETC_LOCATION = ONE_LOCATION ? `${ONE_LOCATION}/etc` : '/etc'
+const mode = process.env.NODE_ENV || 'production'
+const build = process.env.WEBPACK_BUILD_MODE || 'development'
+
+const remotesConfigPath =
+  build === 'production'
+    ? `${ETC_LOCATION}/one/fireedge/sunstone/remotes-config.yaml`
+    : path.resolve(
+        __dirname,
+        '..',
+        '..',
+        '..',
+        'etc',
+        'sunstone',
+        'remotes-config.yaml'
+      )
+
+const remotesConfigContent = fs.readFileSync(remotesConfigPath, 'utf8')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Path traversal vulnerability detected. An attacker could access files outside the intended directory, potentially exposing sensitive information or executing malicious code. Validate and sanitize file paths by using path.normalize() and checking that the resolved path is within the expected directory. Learn More - https://cwe.mitre.org/data/definitions/22.html.